### PR TITLE
Fix README, Compile the output expected

### PR DESCRIPTION
**Pull request by AbandonedCart**
This moves the bintrary instructions as the primary method, but
continues to list the alternatives. More importantly, the user will not
mistakenly follow both sets of instructions as a single process.

### CODE REVIEW

**Comment by FabianTerhorst** [comment]
can you replace compile 'com.pokegoapi:library:0.3' with compile 'com.pokegoapi:PokeGOAPI-library:0.3.2'

**Comment by AbandonedCart** [comment]
@FabianTerhorst I did not change the specific code when submitting the PR. I only moved things around so it wasn't injected into the middle of other instructions.

I am guessing this was the final build after setting up the service, so it was submitted as a separate commit in case it needs to be dropped / edited later.

**Comment by AbandonedCart** [comment]
@FabianTerhorst @Grover-c13 is the base library what is being hosted on bintray? As it stands now, using the information provided, that is what you have as the published dependency.

I have tried to clean it up as best I can, but it is important to watch for overlapping definitions and matching what you intend to get with what you are actually building.

**Comment by Danm72** [comment]
This is going to majorly throw people off.

**Comment by LoungeKatt** [comment]
The entire implementation was too much, too soon.
Something like this is best done either from the beginning, or after things have reached a stable point.

It can't be helped now, though. The changes here are only fixing all the undocumented, mismatched changes that have already been merged.

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 group = 'com.github.aphoh'
```
Comment (FabianTerhorst) [inline build.gradle:1]: also change to com.pokegoapi ?

Comment (AbandonedCart) [inline build.gradle:1]: It doesn't make a difference, but isn't this stuff you should have done in the original PR?

**Comment by LoungeKatt** [comment]
After running a bunch of tests and trying various changes, something in the recent changes has broken local compile. The only reason it wasn't detected before was the copy script I have ignoring the new library because the name wasn't updated to match what it was building.

File: build.gradle (hunk 1 above)
Comment (FabianTerhorst) [inline build.gradle:1]: i don´t think this variable is needed in this file anyway.

**Comment by FabianTerhorst** [comment]
can you remove the changes from the gradle files?

**Comment by FabianTerhorst** [comment]
https://github.com/Grover-c13/PokeGOAPI-Java/pull/448/commits/4bd9f2aeffe69d78e1e063fa698dcec4101b2f67

**Comment by LoungeKatt** [comment]
@FabianTerhorst No. That is an unmerged PR. As I stated when you originally requested to change the version number, I did not want to add that because it was too volatile.

Get your thoughts together, make a final commit that is actually complete, and then I will update the PR to reflect it. Right now, you have another mismatched, disorganized mess.

**Comment by AbandonedCart** [comment]
This now depends on https://github.com/Grover-c13/PokeGOAPI-Java/pull/459 to handle the versioning in the gradle build files and only updates the documentation to match.

**Pull request merged**